### DISCUSSION TITLE
Error after NPM run build - GraphQL Validation Error: MENU_ORDER does not exist in TermObjectsConnectionOrderbyEnum

### DISCUSSION THREAD

**Discussion by beljp** [scottyzen/woonuxt#234] [General]
I'm encountering an issue during the build process of WooNuxt. The error occurs when trying NPM run build

```
`Nuxt 3.13.0 with Nitro 2.9.7

ERROR  GraphQL Document Validation failed with 1 errors;
Error 0: Value "MENU_ORDER" does not exist in "TermObjectsConnectionOrderbyEnum" enum. Did you mean the enum value "TERM_ORDER"?
  at /root/woonuxt/woonuxt_base/app/queries/getProduct.gql:22:35

  ...

Error: GraphQL Document Validation failed with 1 errors;
Error 0: Value "MENU_ORDER" does not exist in "TermObjectsConnectionOrderbyEnum" enum. Did you mean the enum value "TERM_ORDER"?
  at woonuxt_base/app/queries/getProduct.gql:22:35
  ...

```

`Nuxt 3.13.0 with Nitro 2.9.7

ERROR  GraphQL Document Validation failed with 1 errors;
Error 0: Value "MENU_ORDER" does not exist in "TermObjectsConnectionOrderbyEnum" enum. Did you mean the enum value "TERM_ORDER"?
  at /root/woonuxt/woonuxt_base/app/queries/getProduct.gql:22:35

  ...

Error: GraphQL Document Validation failed with 1 errors;
Error 0: Value "MENU_ORDER" does not exist in "TermObjectsConnectionOrderbyEnum" enum. Did you mean the enum value "TERM_ORDER"?
  at woonuxt_base/app/queries/getProduct.gql:22:35
  ...

**Comment by scottyzen** [comment]
Can you try updating [WooGraphQL](https://github.com/wp-graphql/wp-graphql-woocommerce/releases/tag/v0.21.0)

**Reply by beljp** [reply]
seems working sofar! Thanks....it's build now whiteout errors!
